src/ui/components/ShowMoreCard/index.js

```diff
@@ -12,7 +12,7 @@ import Card from 'ui/components/Card';
 import './ShowMoreCard.scss';
 
 
-const MAX_HEIGHT = 100;
+const MAX_HEIGHT = 145;
```

Comment:
We should maybe add a comment saying that it is related to the CSS rule (if I remember correctly).